.github/workflows/docker.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: Extract metadata (tags, labels) for Docker
         id: meta
-        uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
```

Comment:
this commit didn't have the `context` option used below 
<img width="1046" alt="image" src="https://github.com/latticexyz/mud/assets/508855/8fc62421-1543-4d44-9e95-4adb2c039468">